### PR TITLE
Improve the text on the profile page for commercial users

### PR DESCRIPTION
**Pull request by mayhem**
People can get confused as to what waiting status means, so lets spell it out and make the page a little more clear and readable overall.

### CODE REVIEW

**Review by reosarevok — APPROVED** [review]
Seems sensible to me.

File: metabrainz/templates/users/profile.html [hunk 1]
```diff
@@ -75,59 +75,84 @@ <h3>{{ _('Contact information') }}</h3>
...
   {% endif %}
 
   {% if current_user.state in ('active', 'limited') and current_user.good_standing %}
     <h2>{{ _('Data Download') }}</h2>
 
-    {% if not current_user.is_commercial %}
+    <div style="font-size: 13pt">
+      {% if not current_user.is_commercial %}
+        <p>
+           Thank you for registering with us -- we really appreciated it!
```
Comment (atj) [inline metabrainz/templates/users/profile.html:144]:
```suggestion
           Thank you for registering with us -- we really appreciate it!
```

**Comment by mayhem** [comment]
Thanks @atj!

File: metabrainz/templates/users/profile.html [hunk 2]
```diff
@@ -75,59 +75,84 @@ <h3>{{ _('Contact information') }}</h3>
...
+        </p>
+      </div>
     {% else %}
-      <h3>{{ _('Limited data use permission granted') }}</h3>
-      <p>
-        {{ _('You do not currently have permission to use data released under a Creative Commons non-commercial license for commercial purposes.') }}
-      </p>
-      <p>
-        {{ _('Reason:') }}
+      <h3>Limited/no data use permission granted</h3>
```
Comment (MonkeyDo) [inline metabrainz/templates/users/profile.html:97]: it the i18n stuff (`_('…')`) removed on purpose, on this line and a few others below ?

Comment (reosarevok) [reply metabrainz/templates/users/profile.html:97]: Well, we will certainly need i18n support for the parts of the site that will be used for standard users to log in and whatnot, especially once we move to MeB accounts. That said, the business-facing bits being all English make sense to me.

Comment (mayhem) [reply metabrainz/templates/users/profile.html:97]: Its the business and more importantly legal aspects that are super $$$ to fully translate. I'm totally ok, for having the login pages localized.

**Pull request merged**